index.js

```diff
@@ -35,6 +35,24 @@
 
   var IGNORE_CLASS = 'ignore-react-onclickoutside';
 
+  var isSourceFound = function(source, localNode) {
+    if (source === localNode) {
+      return true;
+    }
+    // SVG <use/> elements do not technically reside in the rendered DOM, so
+    // they do not have classList directly, but they offer a link to their
+    // corresponding element, which can have classList. This extra check is for
+    // that case.
+    // See: http://www.w3.org/TR/SVG11/struct.html#InterfaceSVGUseElement
+    // Discussion: https://github.com/Pomax/react-onclickoutside/pull/17
+    if (source.correspondingElement) {
+      return source.correspondingElement.classList.contains(IGNORE_CLASS);
+    }
+    else {
+      return source.classList.contains(IGNORE_CLASS);
+    }
+  };
```

Comment:
function is fine, but we don't need an `else` because of the early return:
```
if (source.correspondingElement) {
  return source.correspondingElement.classList.contains(IGNORE_CLASS);
}
return source.classList.contains(IGNORE_CLASS);
```